static/src/javascripts/projects/common/modules/experiments/tests/adblock-message.js

```diff
@@ -0,0 +1,75 @@
+define([
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/ui/message',
+    'text!common/views/donot-use-adblock.html'
+], function (
+    detect,
+    template,
+    Message,
+    doNotUseAdblockTemplate
+) {
+    return function () {
+        this.id = 'AdBlockMessage';
+        this.start = '2015-03-27';
+        this.expiry = '2015-04-27';
+        this.author = 'Zofia Korcz';
+        this.description = 'Test if the users will disable adblock or at least click support link';
+        this.audience = 0.1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Users will disable adblock or at least click support link';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = 'adblock message, hide, read more';
+        this.idealOutcome = 'Users will disable adblock on theguardian site';
```

Comment:
Makes sense, thought this would be difficult to track. Interesting to see if it has any effect on numbers though. Thanks.